src/util.cpp

```diff
@@ -91,6 +91,22 @@
 #include <openssl/crypto.h>
 #include <openssl/rand.h>
 
+std::vector<std::string> splitByCommas(const std::vector<std::string> &args)
+{
+    std::vector<std::string> result;
+    for (std::string arg : args)
+    {
+        size_t pos;
+        while ((pos = arg.find(',')) != std::string::npos)
+        {
+            result.push_back(arg.substr(0, pos));
+            arg = arg.substr(pos + 1);
```

Comment:
How about also consuming whitespace after a common so "two, three" will work?

Reply:
Was thinking about this, but then you'd also have to enclose the thing with quotation marks on the command line so that it doesn't get broken up into multiple args - I figured it would be an unlikely way to use it.
But maybe when one specifies it in the config file? I'll think about it.